### PR TITLE
Enable u-root

### PR DESCRIPTION
**Pull request by rminnich**
Once the changes are committed for the tmpfs command,
we can start to use u-root.

These changes enable u-root usage for harvey.

Signed-off-by: Ronald G Minnich <rminnich@gmail.com>

### CODE REVIEW

File: lib/namespace [hunk 1]
```diff
@@ -24,6 +24,11 @@ bind -a '#r' /dev
 mount -a /srv/slashn /n
 mount -a /boot /bin
 
+# uroot
+tmpfs /bin/uroot &
```
Comment (sevki) [inline lib/namespace:28]: is tmpfs a directive the namespace file knows? 
or should this be `import tmpfs`

File: sys/src/9/amd64/build.json [hunk 2]
```diff
@@ -1,8 +1,16 @@
 [
 	{
 		"Name": "cpu",
+		"Pre": [
+		    "u-root -o uroot.cpio plan9",
```
Comment (sevki) [inline sys/src/9/amd64/build.json:5]: we need u-root in `bootstrap.sh`

**Review by sevki — CHANGES_REQUESTED** [review]

File: lib/namespace (hunk 1 above)
Comment (rminnich) [reply lib/namespace:28]: tmpfs is the program tmpfs

File: sys/src/9/amd64/build.json (hunk 2 above)
Comment (rminnich) [reply sys/src/9/amd64/build.json:5]: not sure what you mean.

Comment (rminnich) [reply sys/src/9/amd64/build.json:5]: OH! now I know what you mean! 

OK I'll do that first, this PR is not quite right yet anyway, thanks!

**Comment by rminnich** [comment]
ok this one works now.

**Review by gmacd — APPROVED** [review]

**Review by gmacd — APPROVED** [review]

**Comment by rminnich** [comment]
well, SHIT, bootstrap and build are separate
mv $WORK/b001/exe/a.out /home/runner/work/harvey/harvey/linux_amd64/bin/u-root
1900
rm -r $WORK/b001/
ok

**Comment by orangecms** [comment]
#975 is now experimental to see where things are being put. It looks like the Go modules thing and the build tools are not yet cooperating as we'd like. My current approach is to `cp -r` the `$GOPATH/mod/` things to `$GOPATH/src/`. It is ugly, but at least I could get u-root to build a cpio. Now the question is how it would work properly. I will have a look at the Golang docs, hope it'll reveal more details. :)

**Pull request closed without merging**